metaflow/plugins/conda/batch_bootstrap.py

```diff
@@ -46,6 +49,12 @@ def install_conda_environment(env_id, packages):
         'conda create --yes --no-default-packages -p {0} --no-deps {1} >/dev/null 2>&1'.format(os.path.join(os.getcwd(), env_id), ' '.join(packages)),
         'cd {0}'.format(os.getcwd())
     ]
+    if ESCAPE_HATCH_PY is not None:
+        cwd = os.getcwd()
+        generate_trampolines(ESCAPE_HATCH_PY, cwd)
+        print("Conda escape will use %s as the interpreter" % ESCAPE_HATCH_PY)
+    else:
+        print("Could not find a Conda escape interpreter")
```

Comment:
I don't think we should print this statement here. This will show up in user logs and is bound to cause confusion since we wouldn't advertise the escape hatch.